.github/workflows/cd.yml

```diff
@@ -0,0 +1,40 @@
+name: Publish logstash-oss-with-opensearch-plugin
+on:
+  push:
+    tags:
+      - v*
+
+jobs:
+  Build:
+    name: Build and Install logstash-output-opensearch
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout plugin
+        uses: actions/checkout@v1
+
+      - name: Build plugin
+        run: |
+          gem build logstash-output-opensearch.gemspec
+
+      - name: Download and extract logstash
+        run: |
+          mkdir -p prepare-install;
+          mv logstash-output-opensearch-*.gem prepare-install/logstash-output-opensearch.gem;
+          wget --no-verbose https://artifacts.elastic.co/downloads/logstash/logstash-oss-7.13.2-linux-x86_64.tar.gz -P prepare-install/;
```

Comment:
can logstash oss version be made dynamic to download?

Reply:
Ack